symengine/polynomial.cpp

```diff
@@ -8,24 +8,21 @@ namespace SymEngine {
 
 UnivariateIntPolynomial::UnivariateIntPolynomial(const RCP<const Symbol> &var, const unsigned int &degree, map_uint_mpz&& dict) :
      degree_{degree}, var_{var}, dict_{std::move(dict)} {
-
```

Comment:
So change it back to, if I recall correctly, just var and dict as the parameters?